$NetBSD: patch-ae,v 1.2 2011/09/04 01:39:21 dholland Exp $

- recognize NetBSD, OpenBSD, Dragonfly
- use -O3 (maximum level that means anything), not -O6

--- configure.in.orig	1997-08-23 11:35:41.000000000 +0000
+++ configure.in
@@ -58,7 +58,7 @@ case "$OS_TYPE" in
     DEFINES="-DOS_$OS_TYPE -DARCH_$ARCH_TYPE"
     ;;
 
-  Linux|FreeBSD)
+  Linux|FreeBSD|NetBSD|OpenBSD|Dragonfly)
     CC="gcc"
     if test "$ARCH_TYPE" = "i486" ; then
       CFLAGS_ARCH="-m486"
@@ -69,7 +69,10 @@ case "$OS_TYPE" in
     if test "$OS_TYPE" = "FreeBSD" ; then
       OS_TYPE="BSD"
     fi
-    CFLAGS="-O6 -ffast-math -fomit-frame-pointer ${CFLAGS_ARCH}"
+    if test "$OS_TYPE" = "DragonFly" ; then
+      OS_TYPE="BSD"
+    fi
+    CFLAGS="-O3 -ffast-math -fomit-frame-pointer ${CFLAGS_ARCH}"
     DEFINES="-DNO_BYTE_SWAPPING -DOS_$OS_TYPE -DARCH_$ARCH_TYPE" 
     ;;
 
